pkg/vault/contracts/BalancerContractRegistry.sol

```diff
@@ -255,7 +255,14 @@ contract BalancerContractRegistry is IBalancerContractRegistry, SingletonAuthent
     }
 
     /// @inheritdoc IBalancerContractRegistry
-    function isActiveBalancerContract(ContractType contractType, address contractAddress) external view returns (bool) {
+    function isActiveBalancerContract(ContractType contractType, address contractAddress) public view returns (bool) {
```

Comment:
```suggestion
    function isActiveBalancerContract(ContractType contractType, address contractAddress) external view returns (bool) {
```
Can stay external since we have the internal version